src/pages/Details/components/Contents.js

```diff
@@ -84,33 +136,29 @@ const Contents = () => {
             </div>
             <QuantityBox count={count} setCount={setCount} />
           </div>
-          {is_subscribed && <SubscribeOptions />}
+          {isSubscribe ? <SubscribeOptions /> : ''}
```

Comment:
```suggestion
          {isSubscribe && <SubscribeOptions />}
```
이 상황에서는 기존에 사용하셨던 것 처럼 && 연산자를 사용하셔도 됩니다!